wom/src/main/scala/wom/types/WomBooleanType.scala

```diff
@@ -10,6 +10,9 @@ case object WomBooleanType extends WomPrimitiveType {
 
   override protected def coercion = {
     case b: Boolean => WomBoolean(b)
+    case WomOptionalValue(WomBooleanType, Some(WomBoolean(b))) =>
```

Comment:
If we're going to do this should it be everywhere and not just WomBoolean ?
We should check with Commodore that this is not going to break WDL behavior too. The typeclass coercion thingy does seem like the right thing to do

Reply:
...and actually the lack of at least the scientific notation parsing bit of this stuff in float-land seems to be responsible for those broken command lines so I'll be fixing that rn 😄